src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this used anywhere? 

If so, it's a good idea... but I don't see any logging added as part of this review.